pkg/network/sriov/sriov.go

```diff
@@ -19,4 +19,77 @@
 
 package sriov
 
+import (
+	"encoding/json"
+	"fmt"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	virtv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/log"
+
+	"kubevirt.io/kubevirt/pkg/network/namescheme"
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+)
+
 const AliasPrefix = "sriov-"
+
+func CreateNetworkPCIAnnotationValue(networks []virtv1.Network, interfaces []virtv1.Interface, networkStatusAnnotationValue string) string {
+	networkPciMap, err := mapNetworkNameToPCIAddress(networks, interfaces, networkStatusAnnotationValue)
```

Comment:
nit: I suggest to keep the naming consistent:
`networkPciMap` -> `networkPCIMap`
Or
`mapNetworkNameToPCIAddress` -> `mapNetworkNameToPciAddress`

Reply:
used networkPCIMap